pycti/connector/opencti_connector_helper.py

```diff
@@ -1130,8 +1161,38 @@ def send_stix2_bundle(self, bundle: str, **kwargs) -> list:
             )
             return []
 
-        if entities_types is None:
-            entities_types = []
+        # If directory setup, write the bundle to the target directory
+        if bundle_send_to_directory:
+            self.connector_logger.info(
+                self.connect_name + " sending bundle to directory"
+            )
+            bundle_file = time.strftime("%Y%m%d-%H%M%S-") + str(time.time()) + ".json"
```

Comment:
It might be interesting to add the name of the connector when creating the file, so that later, as you mentioned, you can have a connector that ingests all the JSON found in the folder, or with a filter if you only need to ingest data from one or more specific connectors.

Example
`bundle_file = self.connect_name.lower().replace(" ", "_") + "-" + time.strftime("%Y%m%d-%H%M%S-") + str(time.time()) + ".json"`

Reply:
Done thanks